### PR TITLE
Add device name to transaction details

### PR DESCRIPTION
**Pull request by axenov-it**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android
- [ ] n/a

### CODE REVIEW

**Comment by axenov-it** [comment]
image

File: src/components/modals/TransactionAdvanceDetails.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const feeString = {
 
 type OwnProps = {
   bridge: AirshipBridge<null>,
+  deviceName: string,
```
Comment (thehobbit85) [inline src/components/modals/TransactionAdvanceDetails.js:29]: Why do you need to pass the deviceName as a prop and not just call "getDeviceName" in the modal?

File: src/components/scenes/TransactionDetailsScene.js [hunk 2]
```diff
@@ -63,7 +64,8 @@ type State = {
   direction: string,
   bizId: number,
   category: string,
-  subCategory: string
+  subCategory: string,
+  deviceName: string
```
Comment (thehobbit85) [inline src/components/scenes/TransactionDetailsScene.js:68]: Why is the "deviceName" part of the state for this component. 
This is such a waste of code. Just get it directly on the scene above.

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
Fix

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**